internal/testutils/kginkgo.go

```diff
@@ -2,21 +2,25 @@ package testutils
 
 import (
 	"context"
+	"fmt"
 	"time"
 
 	ingressv1alpha1 "github.com/ngrok/ngrok-operator/api/ingress/v1alpha1"
 	ngrokv1alpha1 "github.com/ngrok/ngrok-operator/api/ngrok/v1alpha1"
+	"github.com/ngrok/ngrok-operator/internal/controller"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	DefaultTimeout  = 20 * time.Second
+	DefaultTimeout  = 10 * time.Second
```

Comment:
The thought here was that changing the default timeout to be lower will help keep our tests faster when using the helpers and not specifying a timeout. Tests taking longer will fail and make us look at them closer